src/app/public/modules/select-field/select-field.component.ts

```diff
@@ -196,18 +201,21 @@ export class SkySelectFieldComponent implements ControlValueAccessor {
     }
   }
 
+  public onTouched(): void {
+    this._onTouched();
+    this.blur.emit();
+  }
+
   // Angular automatically constructs these methods.
   /* istanbul ignore next */
   public onChange = (value: any) => { };
-  /* istanbul ignore next */
-  public onTouched = () => { };
```

Comment:
@blackbaud-conorwright this changes the public API of the component and thus would be a breaking change. We may need to rethink how we could do this without a breaking change.

Reply:
The onTouched method? I moved it up to line 204. It has the same input and output, I only changed its contents to also emit the blur event.
How is this breaking, @Blackbaud-TrevorBurch ?

Reply:
:man_facepalming: you're correct. Sorry about that.